examples/templates/aws-linux/main.tf

```diff
@@ -130,7 +130,7 @@ EOT
 resource "aws_instance" "dev" {
   ami               = data.aws_ami.ubuntu.id
   availability_zone = "${var.region}a"
-  instance_type     = "t3.xlarge"
+  instance_type     = "t3.micro"
```

Comment:
This is going to give people a poor experience with cloud workspaces.  I understand the desire to keep our own spend low when these things get orphaned, but I think we need to do this in a way that doesn't cripple end users experience.

We want people to have a good experience with cloud development and get hooked.  A micro is less powerful than my phone.

Reply:
This bring it in line with the AWS windows template. Should be instead update the AWS windows template to match?
Alternatively, we could instead force the user to choose an instance type when creating a workspace via the template.
What do you think?

Reply:
I think modeling it as a variable is a good idea for our example templates.  Give them a reasonable range but keep the default something powerful.  Hopefully our own devs who are just testing, not dogfooding, will choose something small.